bin/import_ww2_db.pl

```diff
@@ -81,7 +81,7 @@ BEGIN
 # Load the webwork3 configuration file.
 
 my $config_file = "$main::ww3_dir/conf/webwork3.yml";
-die "The file $config_file does not exist.  Did you make a copy of it from ww3-dev.dist.yml ?"
```

Comment:
Should we use the same strategy here as in other files, where if webwork3.yml doesn't exist, we use webwork3.dist.yml?

Reply:
Yeah, that sounds good.  I changed that now in this pull request.